src/NHibernate.Test/Async/Hql/EntityJoinHqlTest.cs

```diff
@@ -346,9 +346,9 @@ public async Task NullableEntityProjectionAsync()
 				var withNullManyToOneList = await (session.Query<NullableOwner>().Where(x => x.ManyToOne == null).ToListAsync());
 				var withNullManyToOneJoinedList =
 					await ((from x in session.Query<NullableOwner>()
-							from x2 in session.Query<NullableOwner>()
-							where x == x2 && x.ManyToOne == null && x.OneToOne.Name == null
-							select x2).ToListAsync());
+					from x2 in session.Query<NullableOwner>()
+					where x == x2 && x.ManyToOne == null && x.OneToOne.Name == null
+					select x2).ToListAsync());
```

Comment:
I am not sure why those changes occur. Let get rid of them, if that is not an unstable generation artifact.

Reply:
I think I just made some whitespace changes for #2772 and forgot to regenerate async code.